### PR TITLE
fix(importaccount): add filter function for word list

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Fix filter issue for word list

**Which issue(s) this PR fixes** 🔨

AP-322

### CODE REVIEW

**Comment by phillsatellite** [comment]
Tested: Seems like if user has they're curser around the already selected phrases when they are typing one of the phrases will already be highlighted

https://user-images.githubusercontent.com/93608357/157278343-0433a889-0199-4b94-ae05-41152d13edbc.mov

**Comment by davidchan1219** [comment]
@phillsatellite That is not associated with the existing selected items. As of the position of mouse cursor, highlighting appears.
Do you want me to remove highlighting when the mouse hover around the select menu of that component?

**Comment by phillsatellite** [comment]
> 

Do you mean get rid of the highlight when hovering? I think we should probably keep that

**Comment by davidchan1219** [comment]
@phillsatellite If you keep the highlight, I think the current status is not bad.

**Comment by phillsatellite** [comment]
https://user-images.githubusercontent.com/93608357/157715994-e800bc0d-7a59-4b8f-a105-272c48726fe9.mov

**Comment by stavares843** [comment]
@davidchan1219 please address the above video 🔨

**Comment by davidchan1219** [comment]
@phillsatellite @stavares843 I've just solved the issue. Please check again

**Comment by phillsatellite** [comment]
Tested: Very close! One last little thing to change is when using the keyboard to navigate through the list of phrases it should always start at the top. In screen recording you can see it starts where my mouse is hovering and then goes to the top 

https://user-images.githubusercontent.com/93608357/157922628-23e8867a-1102-43b5-86a7-17269b38cf4c.mov

**Review by phillsatellite — APPROVED** [review]

File: components/ui/TypeHead/TypeHead.vue [hunk 1]
```diff
@@ -65,6 +65,8 @@ export default Vue.extend({
       searchList: [] as Array<string | Object>,
       isFocus: false,
       browseIndex: -1,
+      mouseMove: false,
```
Comment (aewing) [inline components/ui/TypeHead/TypeHead.vue:68]: I might consider renaming these variables to `isActive` and `isMouseTracking` just for consistency and clarity.

**Review by aewing — APPROVED** [review]

**Comment by aewing** [comment]
After speaking with @WanderingHogan there were a few changes that were desired for this ticket.
I have opened a new PR with the discussed changes at https://github.com/Satellite-im/Core-PWA/pull/2422.
Closing this PR for continued discussion there.

**Pull request closed without merging**